ngm/api_views.py

```
@@ -141,3 +146,89 @@ def post(self, request):
+        # Normalize case number to standard format
+        try:
+            case_number = normalize_case_number(case_number_raw)
+        except ValueError as exc:
+            return Response(
+                {"error": str(exc)},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+
+        try:
+            result = get_court_case_details(court_identifier, case_number)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)